.github/workflows/_test_build_fake_prerelease.yaml

```diff
@@ -0,0 +1,84 @@
+name: Test Build *Nix
+
+on:
+  workflow_call:
+    inputs:
+      integration:
+        required: true
+        type: string
+    secrets:
+      OHAI_PFX_CERTIFICATE_BASE64:
+        required: true
+      OHAI_PFX_PASSPHRASE:
+        required: true
+
+jobs:
+  test-build-nix:
+    name: Test binary compilation for all platforms:arch
+    runs-on: ubuntu-latest
+    env:
+      INTEGRATION: ${{ inputs.integration }}
+      REPO_FULL_NAME: ${{ github.event.repository.full_name }}
+      TAG: "v0.0.0"
+    steps:
+      - uses: actions/checkout@v4
+      - run: |
+          git tag "$TAG"
+          if [ -z "$GPG_PASSPHRASE" ]; then
+            echo NO_SIGN=true >> $GITHUB_ENV
+          fi
+      - name: Build all platforms:arch
+        run: make ci/fake-prerelease
+      - name: Upload artifacts for next job
+        uses: actions/upload-artifact@v3
+        with:
+          name: windows-packages
+          path: dist/nri-*.zip
+
+  test-build-windows:
```

Comment:
If I'm not wrong, from the task we defined, the integrations that would need this will be the ones with fake prereleases so I included them under the fake prerelease toggle. Does it make sense?

Reply:
I understand that the idea is to extend this to all ohis , but is something that we can tackle in the future, i guess that default input will be as other `run_windows: true` so adding it will not require modifying all ohi